charts/device-plugin-operator/templates/operator.yaml

```diff
@@ -398,6 +398,16 @@ spec:
   selector:
     control-plane: controller-manager
 ---
+{{- if .Values.privateRegistry.registrySecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.privateRegistry.registrySecret }}
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: {{ printf "{\"auths\": {\"%s\": {\"auth\": \"%s\"}"}}" .Values.privateRegistry.registryUrl (printf "%s:%s" .Values.privateRegistry.registryUser .Values.privateRegistry.registrySecret | b64enc) | b64enc }}
```

Comment:
extra double quote here => }"}}

should be

.dockerconfigjson: {{ printf "{\"auths\": {\"%s\": {\"auth\": \"%\s"}}}"

Reply:
But with this extra double quote I have got the error:

![image](https://github.com/intel/helm-charts/assets/22371448/e86bfc53-ec3a-48b7-964e-63c4af177086)

btw: this extra double quote makes the number of quotes in this string odd (13), so we have unterminated string

Without this extra double quote all is fine:

![image](https://github.com/intel/helm-charts/assets/22371448/72cc96d5-1ea0-4dac-a9e6-0f41d17075a3)

Reply:
@obaranov1 it works, thanks